### PR TITLE
Fix snowflake packaging

### PR DESCRIPTION
**Pull request by anthony2261**

### CODE REVIEW

File: backend/dataline/models/connection/schema.py
Comment (RamiAwar) [inline backend/dataline/models/connection/schema.py:1]: Not sure about this one:
image

Maybe this? (remove the @ from [^@\s])
image

Not matching passwords with special chars. Let's just relax it a bit.

Comment (anthony2261) [reply backend/dataline/models/connection/schema.py:1]: yeah I knew that might be an issue. just not sure if it's ok with sqlalchemy. probably is

Comment (anthony2261) [reply backend/dataline/models/connection/schema.py:1]: done

**Comment by RamiAwar** [comment]
I think you did an oopsie.

Pretty sure this change went in with another PR of yours

**Comment by RamiAwar** [comment]
Double check and close this lol

**Comment by anthony2261** [comment]
This is hurting my head. Some commits went into main already? wth

**Comment by anthony2261** [comment]
Yeah ok I think I know what happened, I must've created this branch off of the snowflake branch instead of from main. Rebased this to main, now there's just one commit which replaces `[^\s]` with `\S` which are equivalent

**Review by RamiAwar — APPROVED** [review]

**Pull request merged**